label_sync/repo_labels.yaml

```diff
@@ -0,0 +1,3527 @@
+Labels:
+  apimachinery:
```

Comment:
is this file purely machine generated? If we want humans to edit it, it might be friendlier to have a format like:

    apimachinery:
    - "bug #ee0701"
    - "cncf-cla: no #ededed"

etc